### PR TITLE
Confirmation Email

### PR DESCRIPTION
**Pull request by seb-wolox**
## Summary

- Added a confirmation email to registration
- All smtp server data is configurable

## Trello Card

https://trello.com/c/wKv8iARy/38-email-de-confirmaci%C3%B3n-al-registrarse

### CODE REVIEW

File: app/controllers/users.js [hunk 1]
```diff
@@ -25,6 +27,17 @@ const save = async (request, response) => {
 
   const userData = newUser.dataValues;
   delete userData.password;
+  transport.sendMail(
```
Comment (gonziesc) [inline app/controllers/users.js]: wrap this in a service

**Review by gonziesc — CHANGES_REQUESTED** [review]

File: app/services/mailSender.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const config = require('../../config').common.email,
+  transport = require('nodemailer').createTransport(config),
+  { registrationHtml } = require('../helper/htmlMessages'),
+  logger = require('../logger');
+
+const sendConfirmationEmail = user =>
```
Comment (gonziesc) [inline app/services/mailSender.js:6]: I'd promisify this, but as u wish

**Review by gonziesc — APPROVED** [review]

File: app/services/mailSender.js (hunk 2 above)
Comment (seb-wolox) [reply app/services/mailSender.js:6]: Hm, when is acceptable to promisify something? I remember that you said it wasn't something too acceptable

Comment (gonziesc) [reply app/services/mailSender.js:6]: this has callbacks, it's asynchronous. What I said was that it is wrong to promisify synchronous things

**Pull request merged**